net/mtxclient/files/patch-CMakeList.txt.diff

```diff
@@ -0,0 +1,10 @@
+--- CMakeLists.txt.orig	2018-07-18 12:32:24.000000000 +0530
++++ CMakeLists.txt	2018-07-18 12:32:33.000000000 +0530
+@@ -50,7 +50,6 @@
+ find_package(MatrixStructs 0.1.0 REQUIRED)
+ find_package(spdlog 0.16.0 CONFIG REQUIRED)
+ find_package(Olm 2 REQUIRED)
+-find_package(sodium REQUIRED)
```

Comment:
Any idea why it can't find libsodium on its own?

Reply:
I'm not an expert on cmake, so I can't be sure, but I think cmak relies on files like FindSodium.cmake or some such, to help it find libraries and set the targets / dependencies appropriately. I'm not sure how to get cmake to see the file if I create it, so I defined the variables as build arguments instead.

Reply:
Perhaps someone who knows more about cmake may pipe up.

Reply:
I've fixed this. For reference, if anyone else sees this, the way to fix it was to make sure `pkgconfig` was installed and then to pass `-DSODIUM_ROOT_DIR=/path/to/sodium/include`